pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -116,7 +116,7 @@ func init() {
 				// BuildControllerFactory.buildDeleteLW
 				{
 					Verbs:     sets.NewString("get", "list", "create", "delete"),
-					Resources: sets.NewString("pods"),
```

Comment:
Also make sure that if you have access by default to create one of the objects that contains a pod template, you'll need access to pods/bind.